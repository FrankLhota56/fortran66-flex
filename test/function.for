	AVG(A, B, C, D) = (A + B + C + D)/4.0
	END
