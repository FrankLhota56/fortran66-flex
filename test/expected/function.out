Line     1: ID(avg)
Line     1: '('
Line     1: ID(a)
Line     1: ','
Line     1: ID(b)
Line     1: ','
Line     1: ID(c)
Line     1: ','
Line     1: ID(d)
Line     1: ')'
Line     1: '='
Line     1: '('
Line     1: ID(a)
Line     1: '+'
Line     1: ID(b)
Line     1: '+'
Line     1: ID(c)
Line     1: '+'
Line     1: ID(d)
Line     1: ')'
Line     1: '/'
Line     1: REAL_LIT(4.000000)
Line     2: EOS
Line     2: END
